srcpkgs/tlf/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'tlf'
+pkgname=tlf
+version=1.4.0
+revision=1
+archs="x86_64 i686"
```

Comment:
is it really glibc/x86 specific? otherwise it's wrong.

Reply:
This is my first package, and I didn't see anything in the manual. How do I specify different makedepends for musl vs glib?

Reply:
it's rarely necessary and should be avoided if possible. The issue here is that `archs="i686 x86_64"` makes it only build on `glibc` and those archs, skipping musl and cross compilation for other targets. So that you should remove this line.

Reply:
Oh I see I was getting confused on glib vs glibc